website/pages/faq.md

```diff
@@ -139,21 +139,11 @@ If you want to challenge a specific user, click their username or click "Find a
 
 Make sure you check your Pokemon's items - items such as Assault Vest, Choice Band, Choice Scarf and Choice Specs will limit your move choices.
 
-### Why is mega x not faster than y?
-
-In Generation 6, turn order is decided before Mega Evolution. If your base form is slower than the opposing Pokemon the turn you Mega Evolve, you will be outsped.
-
-Generation 7 changed the mechanics so that Mega Evolving immediately changes the speed of the evolving Pokemon, meaning that if Pokemon A Mega Evolves and is initially slower then the opposing Pokemon, it will proceed to outspeed it during the same turn.
-
-### How do I mega evolve my Rayquaza?
-
-Rayquaza needs to know the move Dragon Ascent in order for it to Mega Evolve. It is the only Mega Evolution that doesn't require a Mega Stone to Mega Evolve. Also, since Mega Rayquaza is banned in Ubers, you must play a format such as Anything Goes to use it.
-
 ### What is an endless battle?
 
 An endless battle is, as the name implies, a battle that cannot end. The common components of an endless battle are Pokemon that trap the target (via Block, Shadow Tag, or Arena Trap), moves that heal the opponent (usually Heal Pulse), ways to keep your own Pokemon healthy (Slack Off, Recover, or Roost), Leppa Berries to restore the user's lost PP, and continual generation of Leppa Berries (via Recycle).
-
-The opponent is forced to Struggle, and because Struggle can never run out of PP, the battle would go on forever since a player can continuously heal both Pokemon (and prevent them from fainting). Endless battles are banned - if you encounter a "strategy" like this, alert a staff member in the [Help room](https://psim.us/help).
+ 
+The opponent is forced to Struggle, and because Struggle can never run out of PP, the battle would go on forever since a player can continuously heal both Pokemon (and prevent them from fainting). Battles are capped at 1000 turns; if that number of turns is reached, the battle ends and results in a tie. If you encounter a "strategy" like this, alert a global staff member by either sending them a PM or with a help ticket using `/report`.
```

Comment:
Why direct people to report endless battles? The battle terminates after 1000 terms, they're not "endless". Suggest `/offertie` or filing a bug report, what is staff going to do?

Reply:
Agreed, don't report endless battles.

Reply:
I talked to Zodiax (who sent me these changes) and we worked this out.